teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorOptions.java

```diff
@@ -75,6 +75,14 @@
       arity = "1")
   private String validatorExternalSignerUrl = null;
 
+  @Option(
+      names = {"--validators-external-signer-slashing-protection-enabled"},
+      paramLabel = "<BOOLEAN>",
+      description = "Enable internal slashing protection for external signers. Default: true",
+      fallbackValue = "true",
+      arity = "0..1")
+  private boolean validatorExternalSignerSlashingProtectionEnabled = true;
```

Comment:
i would just make this `--validators-internal-slashing-protection-enabled`

Reply:
The pretty much the exact opposite of what it does.  It controls *only* whether the external signers have slashing protection.  The internal signers always have slashing protection enabled and there's no option to disable it.